release.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
Why are we talking about all systems? This script is only for maintainers and we either use Linux Mint (myself) or OS X (mlmorg). I'm pretty sure this is bike shedding and unrelated to the main problem this PR was trying to solve. Please remove this line of code.